app/config/AMIableConfig.scala

```diff
@@ -43,6 +43,21 @@ class AmiableConfigProvider @Inject() (val ws: WSClient, val playConfig: Configu
     playConfig, "stage"
   )
 
+  /**
+    * Cloudwatch metrics are expensive.
```

Comment:
Although CloudWatch can be an expensive service, Cost Explorer shows that we are currently spending < $5 per month for Amiable, so we could simplify here and just push the metrics in `CODE` too?

Reply:
Ah, I didn't actually look at our spend, just assumed it was high! <$5/month isn't that much. Will update to produce metrics from each stage.

It's a strange one though as both CODE and PROD use the same source of data (Prism + AWS accounts), so the metrics will be exactly the same. IIUC each AMIable stage uses Prism PROD too (and in reality Prism CODE === Prism PROD as they scrape the same AWS accounts).